src/app/labeling/qgsmaptoollabel.cpp

```diff
@@ -189,11 +189,10 @@ void QgsMapToolLabel::createRubberBands()
           // instead, just use the boundary of the polygon for the rubber band
           geom = QgsGeometry( geom.constGet()->boundary() );
         }
-        QgsSettings settings;
-        int r = settings.value( QStringLiteral( "qgis/digitizing/line_color_red" ), 255 ).toInt();
-        int g = settings.value( QStringLiteral( "qgis/digitizing/line_color_green" ), 0 ).toInt();
-        int b = settings.value( QStringLiteral( "qgis/digitizing/line_color_blue" ), 0 ).toInt();
-        int a = settings.value( QStringLiteral( "qgis/digitizing/line_color_alpha" ), 200 ).toInt();
+        int r = QgsSettingsRegistryCore::settingsDigitizingLineColorRed.value();
+        int g = QgsSettingsRegistryCore::settingsDigitizingLineColorGreen.value();
+        int b = QgsSettingsRegistryCore::settingsDigitizingLineColorBlue.value();
+        int a = QgsSettingsRegistryCore::settingsDigitizingLineColorAlpha.value();
```

Comment:
would there be a chance to use colors directly?
using a virtual legacy method that could be used if the setting is not yet defined to look somewhere else?

Reply:
but that's not strictly related to this PR.